### PR TITLE
refactor(@ciscospark/eslint-config): use remote eslint-config module

### PR DESCRIPTION
**Pull request by mmaines-cisco**
Remove the packages/node_modules/@ciscospark/eslint-config module and use the remote @ciscospark/eslint-config module by pulling it in as a dependency in package.json. Fix ESLint errors in tooling directory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "test:legacy-browser": "PACKAGE=legacy UPLOAD_DIR=browser grunt test:karma",
     "distsrc": "sed -ibak 's#\"main\": \"dist/index.js#\"main\": \"src/index.js#' packages/node_modules/{*,@ciscospark/*}/package.json && rimraf packages/node_modules/{*,@ciscospark/*}/*bak",
     "srcdist": "sed -ibak 's#\"main\": \"src/index.js#\"main\": \"dist/index.js#' packages/node_modules/{*,@ciscospark/*}/package.json && rimraf packages/node_modules/{*,@ciscospark/*}/*bak",
-    "prepublish": "in-install && npm run --silent link-lint-rules || not-in-install",
+    "prepublish": "in-install",
```
Comment (ianwremmel) [inline package.json]: this whole step should be removed

**Comment by mmaines-cisco** [comment]
@ianwremmel I have made the requested changes. There will probably need to be more when the pipeline is fixed, so please let me know what general changes need to be made there. Thanks

**Comment by ianwremmel** [comment]
are you looking at the issues in the eslint config repo?

**Comment by mmaines-cisco** [comment]
No I need to go do that ! Thanks!

**Comment by ianwremmel** [comment]
no longer relevant

**Pull request closed without merging**